doc/getting-started/building-from-source.rst

```diff
@@ -3,6 +3,46 @@
 Build ACRN from Source
 ######################
 
+Introduction
+************
+
+Following general embedded system programming model, ACRN hypervisor
+does not support one binary for all platforms.
+
+Hypervisor binary is generated based on the platform configuration.
+Different platforms with different configurations shall be set up explicitly
+via Kconfig.
+Instruction can be found in :ref:`getting-started-hypervisor-configuration`.
+
+.. note::
+   A generic configuration named ``hypervisor/arch/x86/configs/generic.config``
+   is provided to help developers to try ACRN easily in the might-be-diversity
+   platform. This generic configuration will likely work for most platforms.
+
```

Comment:
You can enable this configuration by specifing "BOARD=generic" in make command line.